library/src/main/java/com/elvishew/xlog/printer/file/clean/TimeCleanStrategy.java

```diff
@@ -0,0 +1,27 @@
+package com.elvishew.xlog.printer.file.clean;
+
+import java.io.File;
+
+/**
+ * Limit the file life of a max time.
+ */
+public class TimeCleanStrategy implements CleanStrategy {
+
+  private long maxTimeMillis;
+
+  /**
+   * Constructor.
+   *
+   * @param maxTimeMillis the max time the file can keep
+   */
+  public TimeCleanStrategy(long maxTimeMillis) {
+    this.maxTimeMillis = maxTimeMillis;
+  }
+
+  @Override
+  public boolean shouldClean(File file) {
+    long currentTimeMillis = System.currentTimeMillis();
+    long fileTimeMillis = file.lastModified();
+    return (Math.abs(currentTimeMillis - fileTimeMillis) > maxTimeMillis);
```

Comment:
Don't use `Math.abs`, in case user set the phone's time to the past and lost the new log files. `current - lastModified` is enough